core/src/main/scala-3/sttp/tapir/macros/LowPrioSchemaMacrosVersionSpecific.scala

```diff
@@ -0,0 +1,21 @@
+package sttp.tapir
+package macros
+
+import sttp.tapir.Schema.SName
+
+import scala.compiletime.*
+import scala.compiletime.ops.any.IsConst
+
+trait LowPrioSchemaMacrosVersionSpecific:
```

Comment:
should probably be `SchemaCompanionMacrosVersionSpecific`, as `SchemaCompanionMacros` is where the other enumeration-derivation macros are? Also I think we use the "...Extensions" suffix for version-specific code.

Reply:
I don't get the "...Extensions", do you want to call the lowprio trait 'SchemaCompanionMacrosVersionSpecific' or SchemaCompanionMacrosExtensions'?
I sort of applied the same structure zio-json was using, as I created a PR there yesterday.

Reply:
Sorry for not being clear, `SchemaCompanionMacrosExtensions` would follow our naming